containous/maesh
# Update go to v1.15

## What does this PR do?

This PR updates go to 1.15 and traefik to 2.3.0-rc4

Fixes #701 

### How to test it

* check travis
* check semaphore
* build locally
